Exec/RegTests/HITDecay/pelelmex_prob.cpp

```diff
@@ -126,9 +126,9 @@ PeleLM::readProbParm() // NOLINT(readability-make-member-function-const)
   ofs.close();
 
   // Read initial velocity field
-  const int nx = local_prob_parm.input_resolution;
-  const int ny = local_prob_parm.input_resolution;
-  const int nz = local_prob_parm.input_resolution;
+  const size_t nx = local_prob_parm.input_resolution;
```

Comment:
I think this will show clang-tidy issues.

Reply:
I am actually curious about that. We do this in PeleC and it doesn't complain... But I feel like it should.